src/handling.rs

```diff
@@ -177,7 +177,11 @@ impl MessageHandler {
                         }
                     }
                     Message::Broadcast(_, payload) => {
-                        debug!("Received payload with height {:?}", payload);
+                        info!(
```

Comment:
I agree to the content, but maybe this should still be debug level

Reply:
Changed it to debug.